#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'drc_task_common'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()
gen.add("eps_angle", double_t, 0, "", 0.1, 0.0, pi)
exit (gen.generate (PACKAGE, "drc_task_common", "ValveRejector"))
